awsagentprovider/src/main/java/software/amazon/opentelemetry/javaagent/providers/AwsApplicationSignalsCustomizerProvider.java

```diff
@@ -71,6 +71,18 @@ public class AwsApplicationSignalsCustomizerProvider
   private static final String APPLICATION_SIGNALS_EXPORTER_ENDPOINT_CONFIG =
       "otel.aws.application.signals.exporter.endpoint";
 
+  // Histograms must only be exported with no more than 100 buckets per EMF specifications. Per OTEL
+  // documentation,
+  // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#base2-exponential-bucket-histogram-aggregation
+  // max total buckets = max_size*2+1; *2 is because of positive and negative buckets, +1 because of
+  // the zero
+  // bucket. Negatives are not a concern for Application Signals use-case, which only measures
+  // latency, so max_size
+  // of 99 gives total buckets of 100.
+  // MaxScale is being set as the default value as per the OTEL spec.
```

Comment:
Comment is a bit mangled. Can you fix it?
```suggestion
  // Histograms must only be exported with no more than 100 buckets per EMF specifications. Per OTEL documentation,
  // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#base2-exponential-bucket-histogram-aggregation
  // max total buckets = max_size*2+1; *2 is because of positive and negative buckets, +1 because of the zero bucket.
  // Negatives are not a concern for Application Signals use-case, which only measures latency, so max_size of 99 gives
  // total buckets of 100. MaxScale is being set as the default value as per the OTEL spec.
```

Lets also align between Java and Python. Having constants is a good idea, I can try to mimic your impl.